Localizations/KeepingYouAwake/zh-Hant.xcloc/contents.json

```diff
@@ -1,12 +1,12 @@
 {
   "developmentRegion" : "en",
   "project" : "KeepingYouAwake.xcodeproj",
-  "targetLocale" : "zh-Hant-TW",
```

Comment:
Is this an automatic change by Xcode or was it simply not properly set up before?

Reply:
I change to use zh-Hant instead of zh-Hant-TW, as there is no other Hant translation currently.

If someone adds another zh-Hant translation (e.g., zh-Hant-HK), I will update this translation to zh-Hant-TW then.

Reply:
I realized that I forgot to remove the old translation files, removed in 7b60b2d.